osquery/database/database.cpp

```diff
@@ -27,7 +27,7 @@ CLI_FLAG(bool, database_dump, false, "Dump the contents of the backing store");
 
 CLI_FLAG(string,
          database_path,
-         "/var/osquery/osquery.db",
+         OSQUERY_HOME "/osquery.db",
```

Comment:
If data there can be expected to update/write often and it persists reasonably well, go to it.